app/modules/sightings/models.py

```diff
@@ -51,3 +44,9 @@ def get_encounters(self):
     def add_encounter(self, encounter):
         if encounter not in self.encounters:
             self.encounters.append(encounter)
+
+    def delete(self):
+        with db.session.begin():
+            # while self.encounters:  #not going to do this yet!
```

Comment:
Probably needs a TODO to remember

Reply:
this has an _entire google doc_ related to it, so i will definitely be following up on this!  :grinning: